find-replace-stream/main.js

```diff
@@ -0,0 +1,32 @@
+/*
+(c) Copyright 2020 Akamai Technologies, Inc. Licensed under Apache 2 license.
+
+Version: 1.1
+Purpose:  Modify an HTML streamed response by replacing a text string two times across the entire response.
+
+*/
+
+import { ReadableStream, WritableStream } from 'streams'
+import { httpRequest } from 'http-request'
+import { createResponse } from 'create-response'
+import { TextEncoderStream, TextDecoderStream } from 'text-encode-transform'
+import { FindAndReplaceStream } from 'find-replace-stream.js'
+
+export function responseProvider(request) {
+
+	//Get text to be searched for and new replacement text from Property Manager variables in the request object.
```

Comment:
Indentation here looks inconsistent (at least on github) due to tab chars vs spaces. Could you replace the tab indentation on these lines with 4 spaces to match the rest of the file?